src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -238,7 +247,20 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
                   (item) => html`
                     <div>
                       <div>
-                        ${hourly
+                        ${daynight
+                          ? html`
+                              ${new Date(item.datetime).toLocaleDateString(
+                                this.hass!.language,
+                                { weekday: "short" }
+                              )}
+                              ${item.daytime === undefined || item.daytime
```

Comment:
You have a `daytime` property? Why not use that to determine if it is a day/night forecast?

Reply:
Yes, I switched to using this once I realized that the NWS forecast has it and appears to be the only twice daily weather integration at the moment. Hopefully the architecture issue will be resolved and the property will always be there.

Reply:
Well actually we should not merge this, as it is not an official attribute of the weather platform so it should not be implemented on the frontend. We should wait for the architecture issue.

Reply:
Ok. I did make it default to the current behavior (assuming daytime) if `daytime` is undefined.